pkg/reconciler/internal/fake/actionclient.go

```diff
@@ -13,7 +13,8 @@ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 See the License for the specific language governing permissions and
 limitations under the License.
 */
-
+//todo: could we doc why it is required?
```

Comment:
The lint is complaining because it requires that anything that is exported needs to be doc. 
I am not sure if it is a lint issue. Agree that is most important doc what is not "internal" and can be used by its users/consumers. Done any way to pass in the lint.